ds3-sdk/src/main/java/com/spectralogic/ds3client/networking/NetworkClientImpl.java

```diff
@@ -351,5 +347,13 @@ public void close() throws IOException {
                 this.content.close();
             }
         }
+
+        private String getRequestPath() {
```

Comment:
My own personal preference is to make this static, pass in the request, and then push this functionality to a util class so that it can be tested and reused.